site/en/docs/privacy-sandbox/attribution-reporting/multiple-reporters/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Register multiple reporters'
+subhead: >
+  Discover how separate entities can generate and receive attribution reports. 
+description: >
+  Discover how separate entities can generate and receive attribution reports.
+date: 2022-03-31
+updated: 2023-07-17
+authors:
+  - maudn
+---
+
+
+It's possible for two distinct entities to generate and receive attribution reports for a given set of source events, trigger events, or both. To register multiple reporters, you can rely on redirects, or choose other options, as discussed here.
+
+## Registering several triggers using redirects
+
+The following example illustrates the use of redirects to register multiple triggers. In this example, `reporter-1.example` and `reporter-2.example` are two distinct entities looking to generate and receive attribution reports.
+
+At source registration time, the following steps take place:
+
+1. `reporter-1.example` receives a request from the browser.
+1. `reporter-1.example` sends a response that contains the `Attribution-Reporting-Register-Source` header. It could be a regular response but, in this example, it's a redirect to `reporter-2.example` because `reporter-2.example` is another reporter looking to generate and receive attribution reports.
+Note: If `reporter-1` intends to rely completely on `reporter-2` as a measurement service, and does not intend to do any measurement themselves, `reporter-1.example` can omit the `Attribution-Reporting-Register-Source` header.
+1. The browser receives this response. It reads `Attribution-Reporting-Register-Source` and therefore stores a source event, using `reporter-1.example` as the reporting origin. 
+1. Because this request is a redirect, the browser also makes a new request to `reporter-2.example`.
+1. `reporter-2.example` responds with a regular response that contains the `Attribution-Reporting-Register-Source` header. The destination value in this header must be identical to the destination value of the `Attribution-Reporting-Register-Source` header set by `reporter-1.example`. Otherwise, the source will not be registered.
+1. The browser receives this response; it reads `Attribution-Reporting-Register-Source` and stores a source event, using `reporter-2.example` as the reporting origin.
+
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/e4KWsVJR1wCkv4UYYPJj.png", alt="The flow between client and web sites when registering multiple reporters.", width="800", height="460" %}
+
+## Rely on `attributionsrc` instead of redirects as an alternative for source registration 
```

Comment:
I think this header can be "Rely on `attributionsrc` for source registration" because the current phrasing is kind of confusing

Reply:
thanks. changing to Rely on `attributionsrc` for source registration